content/en/docs/v3.6/op-guide/monitoring.md

```diff
@@ -67,6 +67,26 @@ etcd_disk_backend_commit_duration_seconds_bucket{le="0.016"} 406464
 
 Since v3.3.0, in addition to responding to the `/metrics` endpoint, any locations specified by `--listen-metrics-urls` will also respond to the `/health` endpoint. This can be useful if the standard endpoint is configured with mutual (client) TLS authentication, but a load balancer or monitoring service still needs access to the health check.
 
+Since v3.4.29 and v3.5.12, two new endpoints `/livez` and `/readyz` are added. 
```

Comment:
Nit: Lists should be surrounded by blank lines [MD032](https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md032.md)
```suggestion
Since v3.4.29 and v3.5.12, two new endpoints `/livez` and `/readyz` are added.

```